guides/bootstrap/markup/base/form-controls.html

```diff
@@ -81,6 +81,70 @@
           <input type="checkbox" id="checkbox">
         </div>
       </div>
+      <div class="form-group">
```

Comment:
any custom component should be placed in the `patterns/` folder, the `base/` folder is for the default component Bootstrap ships with. Please place the component in its own section (see other custom components)